### PR TITLE
Add jenkinsfile and publish script

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,3 +32,6 @@ gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.0" if Gem.win_platform?
+
+# Use github-pages gem
+#gem "github-pages", group: :jekyll_plugins
```
Comment (oraNod) [inline Gemfile:37]: @karesti and @tristantarrant I think we need the GitHub pages gem for the site. However there is a version incompatibility with Jekyll version 4.0 that is declared in this Gemfile. 

See this issue: https://github.com/github/pages-gem/issues/651 

Building with Jekyll v 4.0 results in error if you include the github-pages gem. If you then downgrade Jekyll to v 3.9.0, all the dependencies match and "bundle update" completes. The problem here is that v3.9.0 does not seem to include the stuff under _plugins. That means the docs aren't fetched.

We could do the docs fetch separately but there is a TagHelpersFilter plugin that I think we might lose too.

Comment (karesti) [reply Gemfile:37]: maybe we can consider using github actions and stop jenkins ? @adelasofia wdyt ?

Comment (oraNod) [reply Gemfile:37]: @karesti I don't think we need github pages after all. I wasn't that sure on Friday and other priorities came up. I've got it working in this PR: https://github.com/infinispan/infinispan.github.io/pull/142

will close this PR

**Comment by oraNod** [comment]
Use https://github.com/infinispan/infinispan.github.io/pull/142 instead

**Pull request closed without merging**